doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1895,7 +1895,7 @@ Fields
 * "client_protocol_version.major", "client_protocol_version.minor": The RFB protocol version agreed by the client.
 * "authentication.security_type": Security type agreed upon in the logged transaction, e.g. ``2`` is VNC auth.
 * "authentication.vnc.challenge", "authentication.vnc.response": Only available when security type 2 is used. Contains the challenge and response byte buffers exchanged by the server and client as hex strings.
-* "authentication.security-result": Result of the authentication process (``OK``, ``FAIL`` or ``TOOMANY``).
+* "authentication.security_result": Result of the authentication process (``OK``, ``FAIL`` or ``TOOMANY``).
```

Comment:
Is this to be changed in master as well ?

Reply:
Yes, it's also an existing inconsistency in the documentation.

Reply:
So, could you fix the documentation in master first please ? (then cherry-pick the doc fix + add a new commit to fix the code in 7)

Reply:
Sure. PR to follow.